doc/modules/ensemble.rst

```diff
@@ -1084,3 +1084,42 @@ Optionally, weights can be provided for the individual classifiers::
 
    >>> eclf = VotingClassifier(estimators=[('lr', clf1), ('rf', clf2), ('gnb', clf3)],
    ...                         voting='soft', weights=[2, 5, 1])
+
+
+ .. _voting_regressor:
+
+Voting Regressor
+================
+
+The idea behind the :class:`VotingRegressor` is to combine conceptually
```

Comment:
Link is broken, please use instead

```suggestion
The idea behind the `VotingRegressor` is to combine conceptually
```

Could you also please do the same for the first occurrence of `VotingClassifier` bove?

Reply:
Still broken